lib/bitcrowd_ecto/repo.ex

```diff
@@ -221,8 +235,17 @@ defmodule BitcrowdEcto.Repo do
     end
   end
 
-  defp ok_tuple_or_not_found_error(nil, error_tag), do: {:error, {:not_found, error_tag}}
-  defp ok_tuple_or_not_found_error(value, _error_tag), do: {:ok, value}
+  defp maybe_rescue_cast_error(opts, callback) do
```

Comment:
is this backwards?
if `{raise_cast_error: false}` then we should rescue, no?

Reply:
maybe add a test where you pass `raise_cast_error: true` ?

Reply:
it's not backwards, but maybe confusing due to the `false` default argument. But it reads "if raise_option, then: code_without_rescue, else: code_with_rescue". Added a test though.

Reply:
ok, yep. that goofed me up. i kept reading it as `get(opts, :raise_cast_error) == false` and couldn't unsee that